cst/plugins/eu.esdihumboldt.cst.functions.numeric/plugin.xml

```diff
@@ -74,6 +74,28 @@
                </bindingCondition>
             </property>
          </targetProperties>
+         <functionParameter
+               description="Generation sequence to use"
+               label="Sequence"
+               maxOccurrence="1"
+               minOccurrence="1"
+               name="sequence">
+            <parameterEnumeration>
+               <enumerationValue
+                     value="overall">
+               </enumerationValue>
+               <enumerationValue
+                     value="type">
+               </enumerationValue>
+            </parameterEnumeration>
+         </functionParameter>
+         <functionParameter
+               description="Starting numerical value for the sequence"
+               label="StartValue"
+               maxOccurrence="1"
+               minOccurrence="1"
```

Comment:
It should be an optional parameter for backwards compatibility.

Reply:
I added now a couple of lines for setting the default value: 
```
<functionParameter
               description="Starting numerical value for the sequence"
               label="StartValue"
               maxOccurrence="1"
               minOccurrence="1"
               name="startValue">
            <parameterBinding
                   class="java.lang.Integer">
            </parameterBinding>      
              <valueDescriptor
                   default="1">
              </valueDescriptor>
         </functionParameter>
```
However, I don't see a different behavior of hs: when the "Generate sequential ID" window pops up, I see, as I have seen before, that the startValue is set on "1". Do I miss something?

Reply:
Solved! By addressing your second comment I see the right behavior with a default value of 1 if I leave the StartValue empty.

Reply:
`minOccurrence` should still be `0`, because the function did already exist before, and any project where it was used before won't have that parameter.